lib/shopify_api/context.rb

```diff
@@ -64,6 +66,7 @@ def setup(
         @api_secret_key = api_secret_key
         @api_version = api_version
         @host_name = host_name
+        @host = T.let(host, T.nilable(String))
```

Comment:
Can `@host` ever be nil here?

Reply:
In the initializer, yes but the default gets set in the `setup` method to build it from the `host_name` if it is nil.

Reply:
Ah, I had misread because of the way GitHub had shown the diff.